libse/TextDraw.cs

```diff
@@ -6,7 +6,7 @@ namespace Nikse.SubtitleEdit.Core
 {
     public static class TextDraw
     {
-        public static void DrawText(Font font, StringFormat sf, GraphicsPath path, StringBuilder sb, bool isItalic, bool isBold, bool isUnderline, float left, float top, ref bool newLine, float leftMargin, ref int pathPointsStart)
+        public static void DrawText(Font font, StringFormat sf, GraphicsPath path, string s, bool isItalic, bool isBold, bool isUnderline, float left, float top, ref bool newLine, float leftMargin, ref int pathPointsStart)
```

Comment:
We can simply pass a string since no build operation is been done inside DrawText.